int/api/cmd/utils.go

```diff
@@ -115,7 +115,7 @@ func CreateReadOnlyResult(result sendOperation.ReadOnlyResult) models.ReadOnlyRe
 				Balance: &models.ChangeSet{
 					Set: oldChange.Update.Balance.Set,
 				},
-				Bytecode:  oldChange.Update.Bytecode,
+				Bytecode:  oldChange.Update.Bytecode.(string),
```

Comment:
I think this is not good.
Here you handle only the case where Bytecode is a string, which is the case when there is no bytecode change and the value is "Keep"
If you want to handle the case where Bytecode is a ChangeSet object, you must detect the type here as following:
		switch v := oldChange.Update.Bytecode.(type) {
		case string:
			newChange.Update.Bytecode = v
		case models.ChangeSet:
			newChange.Update.Bytecode = v
		default:
			newChange.Update.Bytecode = nil
		}

you also need to modify the type of newChange.Update.Bytecode from string to interface{

Reply:
you can achieve this by modify the api yml file directly

Reply:
line 1063 type: string -> type: object